go.mod

```diff
@@ -148,7 +149,7 @@ require (
 	github.com/siderolabs/go-debug v0.4.0
 	github.com/siderolabs/go-kmsg v0.1.4
 	github.com/siderolabs/go-kubeconfig v0.1.0
-	github.com/siderolabs/go-kubernetes v0.2.16
+	github.com/siderolabs/go-kubernetes v0.2.17-0.20241127142321-06f07ab00042
```

Comment:
it'd be nice to have a release (so we go into beta with everything released)

Reply:
yup I can do that